PSReadLine/Cmdlets.cs

```diff
@@ -809,22 +809,27 @@ public object GetDynamicParameters()
     [OutputType(typeof(KeyHandler))]
```

Comment:
Can you please add `DefaultParameterSetName = "FullListing"` to the `Cmdlet(...)` attribute, as follows:
```c#
[Cmdlet("Get", "PSReadLineKeyHandler", DefaultParameterSetName = "FullListing", HelpUri = "https://go.microsoft.com/fwlink/?LinkId=528807")]
```